ext/translation.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, bot):
     async def translate(self, ctx, to_lang: str, *, sentence: str):
         """Translate from one language to another."""
         to_lang = self.bot.clean_content(to_lang).lower()
+        to_lang = to_lang.replace("jp", "ja")
```

Comment:
maybe make this also include cn being aliased to zh-CHS

Reply:
done